core/src/main/java/bisq/core/offer/OfferUtil.java

```diff
@@ -116,7 +116,9 @@ public static Coin getMakerFee(boolean isCurrencyForMakerFeeBtc, @Nullable Coin
      * @param amount
      * @return
      */
-    public static boolean isCurrencyForMakerFeeBtc(Preferences preferences, BsqWalletService bsqWalletService, Coin amount) {
+    public static boolean isCurrencyForMakerFeeBtc(Preferences preferences,
+                                                   BsqWalletService bsqWalletService,
+                                                   Coin amount) {
```

Comment:
please only change stuff that actually changes something. Makes it more cumbersome to review...

Reply:
We all should apply the formatting rules at commit. In IntelliJ there is a checkbox for reorganize import and reformat. If all contributors use that we have same formatting in all committed classes. You can also compare directly in IntelliJ which is much more user friendly as a terminal diff IMO.